docs/src/inverse_problems/petab_ode_param_fitting.md

```diff
@@ -0,0 +1,487 @@
+```
+res_ms = PEtabMultistartOptimisationResult("OptimizationRuns"; which_run="2")
+```
+
+
+## [Events](@id petab_events)
+So far, we have assumed that all experiments, after initiation, run without interference. Experiments where conditions change, or where species are added/removed during the time course, can be represented through events (related to [callbacks](@ref advanced_simulations_callbacks)). In PEtab, an event is represented through the `PEtabEvent` structure. It takes three arguments:
+1. The condition for triggering the event. This can either indicate a point in time, or a boolean condition.
+2. A rule for updating the event's target
+3. The event's target (either a species or parameter).
+
+Here we create an event which adds `0.5` units of `S` to the system at time `5.0`:
+```petab1
+@unpack S = rn
+event1 = PEtabEvent(5.0, S + 0.5, S)
+```
+Here, the first argument is evaluated to a scalar value, in which case it is interpreted as a time point at which the event happens. If we instead want the event to happen whenever the concentration of `S` falls below `0.5` we set the first argument to a boolean condition indicating this:
+```petab1
+event2 = PEtabEvent(S < 0.5, S + 0.5, S)
+```
+
+Whenever we have several events or not, we bundle them together in a single vector which is later passed to the `PEtabODEProblem` using the `events` argument:
+```petab1
+events = [event, event2]
+petab_model = PEtabModel(rn, observables, measurements, params; state_map=u0, events=events)
+```
+
+More details on how to use events, including how to create events with multiple targets, can be found in [PEtab.jl's documentation](https://sebapersson.github.io/PEtab.jl/stable/Julia_event/).
+
+## [Plot recipes](@id petab_plotting)
+There exist various types of graphs that can be used to evaluate the parameter fitting process. These can be plotted using the `plot` command, where the input is either the result of a `calibrate_model` or a  `calibrate_model_multistart` run. To be able to use this functionality, you have to ensure that PEtab.jl [records the optimisation process](@id petab_optimisation_path_recording) by providing the `save_trace=true` argument to the calibration functions.
+
+To, for a single start calibration run, plot, for each iteration of the optimization process, the best objective value achieved so far, run:
+
+```petab1
+plot(res)
+```
+For a multi-start calibration run, the default output is instead a so-called waterfall plot:
+```petab1
+plot(res_ms)
+```
+![petab waterfall plot](../assets/petab_waterfall.svg)
+(for this, and the next plot, we use a multi-start optimisation result from a different model, which yields less trivial optimisation runs than our catalysis one)
+
+In the waterfall plot, each dot shows the final objective value for a single run in the multi-start process. The runs are ordered by their objective values, and colours designate runs in the same local minimum.
```

Comment:
Worth mention that in order to really trust the optmisation results several runs (at least 5) should have converged to the best global minimum?

Reply:
added